components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint.util/src/test/resources/repository/conf/identity/EndpointConfig.properties

```diff
@@ -28,3 +28,4 @@ username.header=UserName
 key.manager.type=SunX509
 trust.manager.type=SunX509
 tls.protocol=TLSv1.2
+google.social.onetap.restricted_browsers=[safari]
```

Comment:
Add a newline at the end of the file

Reply:
This is fixed 4edb01eae541b87003e1683642fa92dd6751bb6d